### DISCUSSION TITLE
[Q&A]  Where can I change the hostname for the agent

### DISCUSSION THREAD

**Discussion by bacus99** [glpi-project/glpi-agent#398] [Q&A] [answered]
### Documentation & bug reporting acknowledgment

Yes, I read it

### Describe your problem

My agent was installed from a Windows server image.  It keeps the agent name of it

Where can I change it?

![image](https://github.com/glpi-project/glpi-agent/assets/11425241/92f994f7-fd8e-4ead-b1db-2826be0140a1)

**Comment by g-bougard** [comment, marked as answer]
Just remove the GLPI-Agent.dump file from the var subfolder you can find into the installation folder. Then restart the agent, it will generate a new agent name based on the current hostname.
You should probably fix the windows server image to remove this file so the first time the agent starts on a new computer based on that image it will create a good agent name.

**Comment by bacus99** [comment]
Works perfect, thank you.  I'll correct the imaging process with my team
